packages/devtools_app/lib/src/screens/inspector/layout_explorer/flex/utils.dart

```diff
@@ -70,8 +68,8 @@ class AnimatedFlexLayoutProperties
           size: Size.lerp(beginProps.size, endProps.size, t),
           realSize: Size.lerp(beginProps.realSize, endProps.realSize, t),
           layoutProperties: AnimatedLayoutProperties(
-            beginProps.layoutProperties,
-            endProps.layoutProperties,
+            beginProps.layoutProperties!,
```

Comment:
this also seems like a sign that the beginProps should have been non-nullable.

Reply:
there are places in code where RenderProperties.layoutProperties are validated for nullness and, in case of null, other properties are used.
This exact method does not do this.
Added TODO.